README.md

```diff
@@ -56,11 +56,11 @@ To install Daytona all you need to do is execute this script:
 
 
 ```bash
-# Install Daytona into ~/bin or ~/.local/bin
-curl -sf -L https://download.daytona.io/daytona/get-server.sh | bash
+# Install Daytona into /usr/local/bin
+curl -sf -L https://download.daytona.io/daytona/get-server.sh | sudo bash
 
-# OR if you want to install Daytona to /usr/loca/bin or /opt/bin
-# curl -sf -L https://download.daytona.io/daytona/get-server.sh | sudo bash
+# OR if you want to install Daytona to some other path where you don`t need sudo
+curl -sf -L https://download.daytona.io/daytona/get-server.sh | DAYTONA_PATH=/home/user/bin bash
```

Comment:
Maybe we should leave this line commented out so the user can just copy/paste this block into their terminal?

Reply:
And now I see they typo in the path string 😂 
Yup that is useful, though still needed for user to adjust variable to his path. So its not just c/p

Reply:
Yeah, comment it out then def